src/unix/core.c

```diff
@@ -35,7 +35,7 @@
 #include <sys/un.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
-#include <limits.h> /* INT_MAX, PATH_MAX */
+#include <limits.h> /* INT_MAX, PATH_MAX, IOV_MAX */
```

Comment:
I'm not following, sorry. On Linux IOV_MAX is already defined to 1024 if you include limits.h. On systems not exposing it that way we query sysconf, otherwise we assume 1024. Taking the Linux example, what value would you set it to, and based on what exactly? Thanks!